app/controllers/collections_controller.rb

```diff
@@ -161,6 +161,21 @@ def delete_item_from_collection
     end
   end
 
+  def delete_document_from_item
```

Comment:
I don't think it'd be an issue, since the ones in JIRA would more-so just be stating the path to use rather than the actual parameter names to use in the routes file. Also the collection_name/collectionId and document_name/filename conceptually refer to same things.